changelog.config.js

```diff
@@ -0,0 +1,88 @@
+const COMPONENT_SPECS = require('genesys-spark-components/build/component-specs/component-specs.json');
+const CHART_COMPONENT_SPECS = require('genesys-spark-chart-components/build/component-specs/component-specs.json');
+
+const componentSpecs = Object.assign(COMPONENT_SPECS, CHART_COMPONENT_SPECS);
+
+const getComponentSpec = function () {
+  let scopeList = ['other'];
+  for (const [key] of Object.entries(componentSpecs)) {
+    scopeList.push(key);
+  }
+  return scopeList;
+};
+
+module.exports = {
+  disableEmoji: true,
+  format: '{type}{scope}: {emoji}{subject}',
+  list: [
+    'test',
+    'feat',
+    'fix',
+    'chore',
+    'docs',
+    'refactor',
+    'style',
+    'ci',
+    'perf'
+  ],
+  maxMessageLength: 110,
+  minMessageLength: 3,
+  questions: ['type', 'scope', 'subject', 'body', 'issues'],
+  scopes: getComponentSpec(),
+  messages: {
+    type: "Review the [contributing guide](./docs/CONTRIBUTING.md) before commiting\nSelect the type of change that you're committing:",
+    scope: 'Select the scope this commit affects:',
+    subject: 'Write a short, imperative mood description of the change:\n',
+    body: 'Provide a longer description of the change:\n ',
```

Comment:
I wonder if just one of these would be sufficient enough or ? 🤔

Reply:
I think there are cases for keeping both `subject` and `body`